updating/updating-cluster-between-minor.adoc

```diff
@@ -17,10 +17,13 @@ Because of the difficulty of changing update channels by using `oc`, use the web
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
+* Ensure all Operators previously installed through Operator Lifecycle Manager (OLM) have been updated to their latest version in their latest channel. This ensures a valid upgrade path for each Operator when the default OperatorHub catalogs switch from the current minor version to the next during the cluster upgrade.
++
+See xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrading installed Operators] for more information.
```

Comment:
@kalexand-rh @codyhoag Adding a prereq in "Updating a cluster between minor versions" for 4.5+. This is particularly important starting with the 4.5 to 4.6 upgrade as we switch from App Registry catalogs to index images:

http://file.rdu.redhat.com/~adellape/100820/upgrade_operators/updating/updating-cluster-between-minor.html#prerequisites

Same treatment for "Updating a cluster within a minor version by using the CLI":

http://file.rdu.redhat.com/~adellape/100820/upgrade_operators/updating/updating-cluster-between-minor.html#prerequisites

Reply:
Updated this to make it one paragraph.